lib/diaspora/mentionable.rb

```diff
@@ -1,23 +1,20 @@
 
 module Diaspora::Mentionable
 
-  # regex for finding mention markup in plain text
-  # ex.
+  # regex for finding mention markup in plain text:
+  #   "message @{user@pod.net} text"
+  # it can also contain a name:
   #   "message @{User Name; user@pod.net} text"
   #   will yield "User Name" and "user@pod.net"
-  REGEX = /(@\{(.+?; [^\}]+)\})/
+  REGEX = /@\{(?:([^\}]+?); )?([^\} ]+)\}/
```

Comment:
@svbergerem, you might want to use this RegEx in your `markdown-it` plugin as well!